lib/slack_interface/request_handler.coffee

```diff
@@ -17,7 +18,9 @@ class SlackInterfaceRequestHandler
 
             # We can store the user who issued the command
             @queuer = request.body['user_name']
-            
+
+            return if @bannedUsers.indexOf(@queuer) > -1
```

Comment:
Does this syntax work? In PHP for example it would need to be ordered differently (pseudocode shown):

```php
if ($bannedUsers->contains($queuer)) return;
```